packages/components/src/tooltip/index.js

```diff
@@ -31,40 +32,81 @@ export const TOOLTIP_DELAY = 700;
 
 const eventCatcher = <div className="event-catcher" />;
 
-const getDisabledElement = ( { eventHandlers, child, childrenWithPopover } ) =>
-	cloneElement(
+/**
+ * Convert a position string to a placement string. This logic
+ * is borrowed from the Popover component, with one key difference:
+ *
+ * Here, the 'left' position translates to the '-start' placement and
+ * the 'right' position translates to the '-end' placement. This ensures
+ * that the ToolTip is placed at the expected left/right location.
+ *
+ * @param {string} position A position string such as 'bottom left'
+ * @return {string} A placement string such as 'bottom-start'
+ */
+const positionToPlacement = ( position ) => {
```

Comment:
Should we add some more unit tests to make sure that this function works properly?

Reply:
Yes, good idea, I'll add some tests tomorrow.